editor.planx.uk/src/ui/editor/MoreInformation/MoreInformation.tsx

```diff
@@ -25,8 +25,9 @@ export const MoreInformation = ({
               multiline
               name="info"
               id="info"
-              value={info}
-              onChange={changeField}
+              value={formik.values.info}
+              // errorMessage={getIn(formik.errors, "info")}
```

Comment:
Comments intentionally left in this file to illustrate the aim of this PR once combined with #4812 

Once the validation is the responsibility of the parent form, we need to pass these errors down into the `RichTextInput` as opposed to bubbling them up from the `RichTextInput` to it's parent `ErrorWrapper`.